packages/core/src/schema/blocks/types.ts

```diff
@@ -220,14 +239,25 @@ export type SpecificBlock<
  *
  */
 
+export type PartialTableCell<
+  I extends InlineContentSchema,
+  S extends StyleSchema = StyleSchema
+> = {
+  type: "tableCell";
+  props?: Partial<TableCellProps>;
+  content?: PartialInlineContent<I, S>;
+};
+
 export type PartialTableContent<
   I extends InlineContentSchema,
   S extends StyleSchema = StyleSchema
 > = {
   type: "tableContent";
   columnWidths?: (number | undefined)[];
+  headerRows?: number;
```

Comment:
How come these are numbers? Isn't it only possible to have 1 of each?

Reply:
We discussed this here: https://github.com/TypeCellOS/BlockNote/issues/1425

But, the idea is that we could in the future support multiple headerRows and headerColumns, but for now we only expose in the UI a single toggle for this. Just gives us more room in the future to implement something more complex if needed